suites/e2e/suite.js

```diff
@@ -63,19 +64,19 @@ module.exports = {
 	title: 'Testbot Diagnostics',
 	run: async function (test) {
 		// The worker class contains methods to interact with the DUT, such as flashing, or executing a command on the device
-		const Worker = this.require('common/worker');
+		// const Worker = this.require('common/worker');
 		// The balenaOS class contains information on the OS image to be flashed, and methods to configure it
-		const BalenaOS = this.require('components/os/balenaos');
+		// const BalenaOS = this.require('components/os/balenaos');
 		// The `BalenaSDK` class contains an instance of the balena sdk, as well as some helper methods to interact with a device via the cloud.
-		const Balena = this.require('components/balena/sdk');
+		// const Balena = this.require('components/balena/sdk');
 		await fse.ensureDir(this.suite.options.tmpdir);
 
 		// The suite contex is an object that is shared across all tests. Setting something into the context makes it accessible by every test
 		this.suite.context.set({
-			utils: this.require('common/utils'),
+			utils: utils,
 			sshKeyPath: join(homedir(), 'id'),
 			sshKeyLabel: this.suite.options.id,
-			sdk: new Balena(this.suite.options.balena.apiUrl, this.getLogger()),
+			sdk: new Sdk(this.suite.options?.balena?.apiUrl, this.getLogger()),
```

Comment:
I know that this is called sdk in the helpers package - bit isn't it a little bit more than just the SDK? We have some CLI uses in there, and some extra functions that are a composite of various sdk calls + CLI uses

Reply:
It is but I felt it was more confusing to import a component called Balena which is a file called sdk.js and initialize it as SDK. 

Why confuse when we can straight away be called it SDK? 
Coming over to what's inside of it, I think that should be broken down further in the helper's package to improve our consistency. The SDK helper should only have SDK methods. What do you think? 

(This is the time and place we can semantically improve our nomenclature as well as structures) Happy to get more input on helpers package before merging this so we can get the best changes in there.